components/prism-powershell.js

```diff
@@ -9,26 +9,38 @@ Prism.languages.powershell = {
 			lookbehind: true
 		}
 	],
-	'string': {
-		pattern: /("|')(`?[\w\W])*?\1/m,
-		inside: {}
-	},
+	'string': [
+		{
+			pattern: /"(`?[\w\W])*?"/m,
+			inside: {
+				'function': {
+					pattern: /[^`]\$\(.*?\)/,
+					inside: {}
+				}
+			}
+		},
+		/'([\w\W])*?'/m
```

Comment:
You don't need the parentheses here. The `m` flag is not needed either.

Actually, I think this could be simplified as `/'[^']*'/`, even if we might want to handle the escaping better. (See my comment in the string test).